indico/modules/events/__init__.py

```diff
@@ -205,3 +205,18 @@ def _event_cloned(old_event, new_event, **kwargs):
     new_event.contact_title = old_event.contact_title
     new_event.contact_emails = old_event.contact_emails
     new_event.contact_phones = old_event.contact_phones
+
+
+@signals.event.updated.connect
+def _patch_google_wallet_class(sender, obj=None, **kwargs):
```

Comment:
why is this in `events` and not `events/registration`?

Reply:
Well, because we will update the Google Wallet class if there is a change in the Event data, so I consider it is related to the Event and not to Registration... but it will be ok for me if you prefer to move it elsewhere :)

Reply:
The signal is from event but the stuff you are doing is related to registrations -> belongs in the registration module